modules/admission-webhook-about.adoc

```diff
@@ -0,0 +1,52 @@
+* During the admission process, a mutating admission controller can invoke mutating admission webhooks to perform tasks, such as injecting affinity labels.
+
+//Future xref - * At the end of the admission process, a validating admission controller can invoke xref:../architecture/admission-controllers.adoc#validating-admission-webhooks[validating admission webhooks] to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+* At the end of the admission process, a validating admission controller can invoke validating admission webhooks to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+
+When an API request comes in, a mutating or validating admission controller uses the list of external webhooks in the configuration and calls them in parallel:
+
+* If all of the webhooks approve the request, the admission chain continues.
+
+* If any of the webhooks deny the request, the admission request is denied and the reason for doing so is based on the first denial.
+
+* If more than one webhook denies the admission request, only the first is returned to the user.
+
+* If there is an error encountered when calling a webhook, that request is either denied or the webhook is ignored.
+
+//Future xrefs - The communication between the admission controller and the webhook server must use TLS. Generate a certificate authority (CA) certificate and use the certificate to sign the server certificate that is used by your webhook server. The PEM-encoded CA certificate is supplied to the admission controller using a mechanism, such as xref:../authentication/certificates/service-serving-certificate.adoc#service-serving-certificate[service serving certificate secrets].
+The communication between the admission controller and the webhook server must use TLS. Generate a CA certificate and use the certificate to sign the server certificate that is used by your webhook server. The PEM-encoded CA certificate is supplied to the admission controller using a mechanism, such as service serving certificate secrets.
+
+The following diagram illustrates the sequential admission chain process within which multiple webhook servers are called.
+
+.API admission chain with mutating and validating webhooks
+image::api-admission-chain.png["API admission stage", align="center"]
+
+An example use case for admission webhooks is syntactical validation of resources. For example, in an infrastructure where all pods must have a common set of labels, a webhook can inject labels and another webhook can verify that labels are as expected. {product-title} would subsequently schedule pods that include required labels and reject those that do not.
```

Comment:
syntactical? Which syntax?

Reply:
I have removed the word syntactical now and it reads much clearer.